From 6bd7447b9b48a5d8db9d8648c71dbe699164579f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jonas=20Dre=C3=9Fler?= <verdre@v0yd.nl>
Date: Mon, 7 Oct 2019 15:00:19 +0200
Subject: [PATCH 1/2] magnifier: Use own showSystemCursor() instead of
 set_pointer_visible()

We already have our own function to show the system cursor, use it!

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/754
---
 js/ui/magnifier.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/magnifier.js b/js/ui/magnifier.js
index 0ec30ee00..0a935edef 100644
--- a/js/ui/magnifier.js
+++ b/js/ui/magnifier.js
@@ -169,7 +169,7 @@ var Magnifier = class Magnifier {
         // Make sure system mouse pointer is shown when all zoom regions are
         // invisible.
         if (!activate)
-            this._cursorTracker.set_pointer_visible(true);
+            this.showSystemCursor();
 
         // Notify interested parties of this change
         this.emit('active-changed', activate);
-- 
2.23.0

